internal/webhook/sparkpod_defaulter.go

```diff
@@ -475,21 +475,27 @@ func addSchedulerName(pod *corev1.Pod, app *v1beta2.SparkApplication) error {
 	return nil
 }
 
+func setPodPriorityClassName(pod *corev1.Pod, priorityClassName *string) {
+	if priorityClassName != nil && *priorityClassName != "" {
+		pod.Spec.PriorityClassName = *priorityClassName
+		pod.Spec.Priority = nil
+		pod.Spec.PreemptionPolicy = nil
+	}
+}
+
```

Comment:
Perhaps we could merge the logic of `setPodPriorityClassName` into `addPriorityClassName`.

Reply:
@ChenYi015 Already adapted merge the logic of setPodPriorityClassName into addPriorityClassName, thanks for ur suggestions